google-storage/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "content-provider",
+  "name": "@nteract/google-content-provider",
   "version": "1.0.0",
   "description": "A collection of content providers for accessing notebooks in different storage locations",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js",
```

Comment:
We'll also need a build script to transpile the TypeScript to JavaScript when publishing.